Source/ViewModel/MLCardFormViewModel.swift

```diff
@@ -191,15 +191,19 @@ final class MLCardFormViewModel {
         }
     }
     
+    
     func groupIndexOfCardFormField(_ cardFormField: MLCardFormField) -> Int? {
         let fieldId = cardFormField.property.fieldId()
-        let index = cardFormFields?.firstIndex(where: {
+        guard let index = cardFormFields?.firstIndex(where: {
             $0.first(where: {
                 $0.property.fieldId() == fieldId
             }) != nil
-        })
+        }) else { return nil }
         
-        return index
+        if fieldId == MLCardFormFields.expiration.rawValue {
+            return index
+        }
+        return Int(index + 1)
```

Comment:
i only have one question, are we starting the group of cards from position 1? i'm just curios about adding `1`

Reply:
@rgrajalesmeli I'm changing the way I call the next screen.
guard let index = viewModel.groupIndexOfCardFormField(cardFormField),
            let cardFieldCollectionView = cardFieldCollectionView else { return }
        guard index != currentCellIndex() else {
            return
        }

I'm adding the +1 to the index so I can get the next element in the collection.
I tried to follow the existing logic in the lib.